server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/NumericTermsAggregator.java

```diff
@@ -164,17 +164,18 @@ public void collectDebugInfo(BiConsumer<String, Object> add) {
                 BucketOrdsEnum ordsEnum = bucketOrds.ordsEnum(owningBucketOrds[ordIdx]);
                 Supplier<B> emptyBucketBuilder = emptyBucketBuilder(owningBucketOrds[ordIdx]);
                 while (ordsEnum.next()) {
+                    long docCount = bucketDocCount(ordsEnum.ord());
```

Comment:
I flip the `if` statement here and moved the spare initialization down to right before I need it to line up with the way the `map` version works.